intro-HPC/ch_quick_reference_guide.tex

```diff
@@ -3,35 +3,35 @@ \chapter{\hpc Quick Reference Guide}
 
 \begin{tabular}{|l|l|} \hline
 \multicolumn{2}{|c|}{\strong{Login}} \\ \hline
-Login             & ssh $<$vsc-account$>$@\loginnode \\ \hline
+Login             & ssh <{}vsc-account>{}@\loginnode \\ \hline
```

Comment:
Why not use `\userid` here (and mention clearly to replace it with your own VSC login name)?